src/adagrams.js

```diff
@@ -1,6 +1,139 @@
 const Adagrams = {
+  lettersDistribution: [
```

Comment:
This data structure works, but it's not very DRY and would be tricky to change. For example, suppose I said you had the wrong number of "I"s - you would have to do a lot of counting to solve the problem.

Instead you might store a hash of letter frequencies like this

```
letterQuantities: {
  A: 9, B: 2, ...
}
```